### ISSUE TITLE
Investigate aria-dsiabled vs disabled

### ISSUE THREAD

**Issue by bethcollins92** [canonical-web-and-design/vanilla-framework#4299]
There are some accessibility issues with using the native browser `disabled` attribute, as it hides the element from Assistive Tech.

The general advice is to use aria-disabled attribute but this doesn't come with the functionality of actually making the button disabled, or any of the styles. We need to look into how important it is to move to using aria-disabled, as it would be a lot of work. 

https://a11y-101.com/development/aria-disabled

**Comment by bethcollins92** [comment]
Another interesting article: https://css-tricks.com/making-disabled-buttons-more-inclusive/

This article shares a lot of the same advice, that using `disabled` is the worst option, aria-disabled is second best and better still is to keep the button enabled and provide feedback on why the form can't be submitted

**Comment by clagom** [comment]
We discussed this issue in our Design Assembly today. It's a large task that has multiple repercussions across all our websites and applications. We are going to take some time to inform more ourselves on this matter, then I would like to organise a combined workshop across disciplines.

**Comment by bartaz** [comment]
Triage: this would be a high effort to do it on current architecture. This should be part of accessibility considerations for new architecture.

**Comment by bartaz** [comment]
Related to: https://github.com/canonical/vanilla-framework/issues/4942
